Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ruby-sys"
-version = "0.2.13"
+version = "0.3.0"
```

Comment:
@d-unseductable I suggest adding this to (a currently non-existent) `CONTRIBUTING.md` (I need to add something about version bumping in all of my projects too...)